aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -1129,6 +1129,12 @@ TORCH_LIBRARY_IMPL(aten, Batched, m) {
   BINARY_POINTWISE_VA(rsub, Scalar);
   BINARY_POINTWISE(mul);
   BINARY_POINTWISE(div);
+  {
+    using Binop = Tensor (*)(const Tensor&, const Tensor&, std::string);
```

Comment:
@rzou would you take a look here?

Reply:
@mruberry I think you got the wrong user. Was that meant to be @zou3519?

Reply:
It was, thanks @peterbell10. Darn autocomplete!

cc @zou3519

Reply:
this lgtm!